packaging/helm/skipper-aws/templates/external-dns/deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: external-dns
+  namespace: kube-system
+  labels:
```

Comment:
It might also be useful to include the chart version as a label (`{{ .Chart.Version }}`)

Reply:
makes sense, will change it

Reply:
fixed